packages/amplify-cli/src/project-config-version-check.ts

```diff
@@ -1,7 +1,8 @@
 import * as path from 'path';
 import * as fs from 'fs-extra';
+import * as inquirer from 'inquirer';
 import { Context } from './domain/context';
-import inquirer from './domain/inquirer-helper';
+import { ConfirmQuestion } from 'inquirer';
```

Comment:
Do we need this, since you're importing inqurier on line number 3

Reply:
Yes, since this is coming from inquirer package the other one is some helper, did not refactor anything in this PR, just conversion.